.travis.yml

```diff
@@ -29,10 +29,10 @@ before_script:
 
 script:
   - cd vms
-  - python manage.py makemigrations auth volunteer administrator organization event job shift registration
-  - python manage.py migrate --noinput --traceback --settings=vms.settings
-  - coverage run --source='.' manage.py test -v 2
-  - coverage report -m
```

Comment:
Why remove this? It is actually helpful as it gives report of the lines missed in tests alongside the corresponding file even if the coveralls fails to send a report.

Reply:
I haven't removed it. Lemme check